core/color.h

```diff
@@ -51,8 +51,8 @@ struct Color {
 	bool operator==(const Color &p_color) const { return (r == p_color.r && g == p_color.g && b == p_color.b && a == p_color.a); }
 	bool operator!=(const Color &p_color) const { return (r != p_color.r || g != p_color.g || b != p_color.b || a != p_color.a); }
 
-	uint32_t to_32() const;
-	uint32_t to_ARGB32() const;
+	uint32_t to_rgba32() const;
+	uint32_t to_argb32() const;
 	uint32_t to_ABGR32() const;
```

Comment:
Please rename "to_ABGR32" too. Thank you. :)

Reply:
I saw this too but I wasn't sure if I should because it was just recently added, like a week ago and it doesn't seem to be hooked up to scripting yet. Seems to be only used for image_loader_svg.cpp so far.

Reply:
I think it should be fixed anyway. That method was added after #8830 had been closed so it probably wasn't checked.

Reply:
Updated. :)